lib/modules/platform/gitea/utils.ts

```diff
@@ -137,3 +137,33 @@ export function toRenovatePR(data: PR, author: string | null): Pr | null {
     hasAssignees: !!(data.assignee?.login ?? is.nonEmptyArray(data.assignees)),
   };
 }
+
+/**
+ * Check if a repository is usable.
+ * A repo isn't usable:
+ * - if it's a mirror
+ * - if we don't have push permissions
+ * - if it doesn't have pull requests
```

Comment:
```suggestion
 * A repo is usable when:
 * - it's a source repo (mirrors do not work)
 * - we have push permission to the repo
 * - the repo supports pull requests
```

We should focus on what we need for a repository to be usable.

Focusing on what needs to be _broken_ for the repository to _not_ work makes the comment harder to understand.

Please double check if the "source repo" bit is correct. Also, do we mean "fork" or "mirror"?